### PR TITLE
Update sailtrack-ground fork to reflect changes in sailtrack-core v1.3.0

### LINKED ISSUE #13: Update sailtrack-ground fork to reflect changes in sailtrack-core v1.3.0
Observing the recent updates in the sailtrack-core repository, it appears that there are modifications that could benefit sailtrack-ground.
I propose to synchronize sailtrack-ground with sailtrack-core to incorporate the latest changes and improvements. This synchronization will ensure that sailtrack-ground remains aligned with the broader project's advancements.

### PR DESCRIPTION
**Pull request by Magform**

### CODE REVIEW

File: .build/sailtrack-build [hunk 1]
```diff
@@ -1,31 +1,20 @@
...
-OUTPUT_IMG_NAME=$2
-
-# Download and extract image
-wget -N "https://dietpi.com/downloads/images/$DIETPI_IMG_NAME.7z" -P /tmp/sailtrack-build || exit 1
-7z e -aoa -o"/tmp/sailtrack-build/$DIETPI_IMG_NAME" "/tmp/sailtrack-build/$DIETPI_IMG_NAME.7z"
+# `sailtrack-build` - Script that builds the SailTrack OS image starting from a DietPi image.
+
+debian_release_name="Bookworm"
+dietpi_image_name="DietPi_RPi-ARMv8-$debian_release_name"
+sailtrack_image_name="SailTrack-Core_RPi-ARMv8-$debian_release_name"
```
Comment (matteocarnelos) [inline .build/sailtrack-build]:
```suggestion
sailtrack_image_name="SailTrack-Ground_RPi-ARMv8-$debian_release_name"
```

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        uses: actions/checkout@v3
+
+      - name: Run SailTrack-Build
+        run: sudo ./.build/sailtrack-build
+
+      - name: Upload artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: SailTrack-Core_RPi-ARMv8-Bookworm.7z
+          path: SailTrack-Core_RPi-ARMv8-Bookworm.7z
```
Comment (matteocarnelos) [inline .github/workflows/build.yml]:
```suggestion
          name: SailTrack-Ground_RPi-ARMv8-Bookworm.7z
          path: SailTrack-Ground_RPi-ARMv8-Bookworm.7z
```

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -8,18 +8,15 @@ on:
...
 
       - name: Run SailTrack-Build
-        run: sudo ./.build/sailtrack-build ${{ env.DIETPI_IMG_NAME }} ${{ env.OUTPUT_IMG_NAME }}
+        run: sudo ./.build/sailtrack-build
 
       - name: Publish release
         uses: softprops/action-gh-release@v1
         with:
-          files: ${{ env.OUTPUT_IMG_NAME }}.7z
+          files: SailTrack-Core_RPi-ARMv8-Bookworm.7z
```
Comment (matteocarnelos) [inline .github/workflows/publish.yml]:
```suggestion
          files: SailTrack-Ground_RPi-ARMv8-Bookworm.7z
```

**Review by matteocarnelos — CHANGES_REQUESTED** [review]
Well done! However, you left some "Core" references somewhere. I should have highlighted all of them, but consider doing a global search with your IDE to find all the left occurrences,  if any.

**Comment by Magform** [comment]
I should have removed all "Core" references

**Review by matteocarnelos — APPROVED** [review]
LGTM

**Pull request merged**